Prepare README.md and package.json files for v3 release (#141)
developit/htm: README.md

> This meant giving up JSX, and the closest alternative was [Tagged Templates]. So, I wrote this library to patch up the differences between the two as much as possible. As it turns out, the technique is framework-agnostic, so it should work great with most Virtual DOM libraries.

Reviewer: ```suggestion
This meant giving up JSX, and the closest alternative was [Tagged Templates]. So, I wrote this library to patch up the differences between the two as much as possible. The technique turns out to be framework-agnostic, so it should work great with any library or renderer that works with JSX.
```

> As of 3.0.0, `htm` is stable, well-tested and ready for production use.

Reviewer: ```suggestion
`htm` is stable, fast, well-tested and ready for production use.
```
